src/main/java/org/folio/innreach/domain/entity/CentralServer.java

```diff
@@ -125,14 +132,18 @@
       "cs.centralServerCode, " +
       "cs.centralServerCredentials.centralServerKey, " +
       "cs.centralServerCredentials.centralServerSecret" +
-    ") FROM CentralServer AS cs ";
+      ") FROM CentralServer AS cs ";
 
   public static final String FETCH_CONNECTION_DETAILS_BY_ID_QUERY = FETCH_CONNECTION_DETAILS_QUERY + FETCH_BY_ID_POSTFIX;
   public static final String FETCH_CONNECTION_DETAILS_BY_CENTRAL_CODE_QUERY = FETCH_CONNECTION_DETAILS_QUERY + FETCH_BY_CENTRAL_CODE_POSTFIX;
 
   public static final String FETCH_RECALL_USER_BY_ID_QUERY_NAME = "CentralServer.fetchRecallUser";
   public static final String FETCH_RECALL_USER_BY_ID_QUERY = "SELECT cs FROM CentralServer AS cs LEFT JOIN FETCH cs.innReachRecallUser " + FETCH_BY_ID_POSTFIX;
 
+  public static final String VALIDATE_AGENCY_LIBRARIES_QUERY_NAME = "CentralServer.validateAgencyLibraries";
```

Comment:
I'd put this to LocalAgency entity, because agency connected to library directly not CS